packages/number-input/src/Input/Input.tsx

```diff
@@ -162,23 +163,23 @@ export const Input = React.forwardRef<HTMLInputElement, InputProps>(
       }
     };
 
-    const handleOnFocus = () => {
+    const handleFocusContainer = () => {
```

Comment:
Could we resolve this by listening to the focus/blur event that bubbles up from the input or button, instead of adding a listener to each element?

Reply:
thanks for this feedback, I like this approach better